src/test/frontend/replicasetdetail/sortedheader_controller_test.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SortedHeaderController from 'replicasetdetail/sortedheader_controller';
+import {UPWARDS, DOWNWARDS} from 'replicasetdetail/sortedheader_controller';
+
+describe('Sorted header controller', () => {
+  /**
+   * Sorted header controller.
+   * @type {!SortedHeaderController}
+   */
+  let ctrl;
+
+  beforeEach(() => { ctrl = new SortedHeaderController(); });
+
+  it('should switch sorting order after clicking same column twice', () => {
+    // given "selected-column" is sorted upwards
+    ctrl.currentlySelectedOrder = UPWARDS;
+    ctrl.currentlySelectedColumn = "selected-column";
+
+    // when user clicks it once more
+    ctrl.columnName = "selected-column";
+    ctrl.changeSorting();
+
+    // then it should be sorted downwards
+    expect(ctrl.currentlySelectedColumn).toEqual(ctrl.columnName);
+    expect(ctrl.currentlySelectedOrder).toEqual(DOWNWARDS);
+    expect(ctrl.isArrowUp()).toEqual(false);
+    expect(ctrl.isArrowDown()).toEqual(true);
+  });
+
+  it('should switch sorting column after clicking second column', () => {
+    // given "first-column" is sorted upwards
+    ctrl.currentlySelectedOrder = UPWARDS;
+    ctrl.currentlySelectedColumn = "first-column";
+
+    // when user clicks "second-column"
+    ctrl.columnName = "second-column";
+    ctrl.changeSorting();
+
+    // then it should be sorted by "second-column" upwards
+    expect(ctrl.currentlySelectedColumn).toEqual(ctrl.columnName);
```

Comment:
I think it should be left as it is, because firstly if in the future we would like to change this string then we will change the value only in one place. I know, that writing `"first-column"` is easier to read than `ctrl.columnName`, but in this case I'm against doing it. Secondly test is very simple and it's easy to notice now, that expected value after `ctrl.changeSorting()` invocation is `ctrl.columnName` not some `"first-column"` string.